src/components/views/rooms/wysiwyg_composer/DynamicImportWysiwygComposer.tsx

```diff
@@ -39,8 +39,24 @@ export const dynamicImportSendMessage = async (
 };
 
 export const dynamicImportConversionFunctions = async (): Promise<{
+    /**
+     * Creates a rust model from rich text input (html) and uses it to generate the plain text equivalent (which may
+     * contain markdown). The return value must be used to set `.innerHTML` for consistency in handling linebreaks.
```

Comment:
```suggestion
     * contain markdown). The return value must be used to set `.innerHTML` (rather than `.innerText`) to
     * ensure that HTML entities are correctly interpreted, and to prevent newline characters being turned into `<br>`.
```

is that right? Does `richToPlain` actually return newline characters?

Reply:
Yes, that's correct